sshd-core/src/main/java/org/apache/sshd/client/channel/ChannelSession.java

```diff
@@ -48,6 +53,7 @@ public class ChannelSession extends AbstractClientChannel {
 
     private CloseableExecutorService pumperService;
     private Future<?> pumper;
+    private Map<String, Object> env = new LinkedHashMap<>();
```

Comment:
Should be `final` - unless there is some reason to make it mutabel (?)

Reply:
agree, no reasons to leave it mutable. Fixed